src/main/kotlin/lotto/domain/enums/RANK.kt

```diff
@@ -0,0 +1,13 @@
+package lotto.domain.enums
+
+import lotto.domain.Prize
+
+enum class RANK(
```

Comment:
enum class의 경우에도 Camel Case로 명명해주세요 🙏
```suggestion
enum class Rank(
```